README.md

```diff
@@ -2,40 +2,74 @@
 
 ![](https://img.shields.io/badge/Kubernetes-1.15-326de6.svg) ![Build Status](https://travis-ci.org/ubuntu/microk8s.svg?branch=master)
 
-<img src="/docs/images/certified_kubernetes_color-222x300.png" align="right" width="200px">Kubernetes in a [snap](https://snapcraft.io/) that you can run locally.
+<img src="/docs/images/certified_kubernetes_color-222x300.png" align="right" width="200px">Kubernetes in a [snap](https://snapcraft.io/).
 
-You can install MicroK8s with the latest stable upstream Kubernetes release with:
+Dead simple to install, fully featured, always fresh, upstream Kubernetes available in 42 Linux distributions. Prefect for:
```

Comment:
Instead of - Dead simple to install, fully featured, always fresh, upstream Kubernetes
Prefer - Dead simple to install, fully featured, always current with upstream Kubernetes,